scripts/curate_data_axondeepseg_tem.py

```diff
@@ -0,0 +1,162 @@
+                path_file_out = os.path.join(path_sub_id_dir_out, sub_bids_full + der[file])
+                create_path_sub_id_dir_out(path_sub_id_dir_out)
+            shutil.copy(path_file_in, path_file_out)
+
+    sub_list = os.listdir(output_data)
+    sub_list.remove('derivatives')
+    sub_list.sort()
+
+    # Write additional files
+    import csv
+    participants = []
+    samples = []
+    for subject in sub_list:
+        row_sub = []
+        row_sub.append(subject)
+        participants.append(row_sub)
+        list_samples = []
+        for sample in os.listdir(os.path.join(output_data, subject, 'microscopy')):
+            if sample.endswith('.png'):
+                list_samples.append(sample)
+        list_samples.sort()
+        for file_sample in list_samples:
+            row_sub_samples = []
+            row_sub_samples.append(subject)
+            row_sub_samples.append(file_sample.split('_')[1])
+            row_sub_samples.append('tissue')
+            samples.append(row_sub_samples)
+
+    # Create participants.tsv
+    with open(output_data + '/participants.tsv', 'w') as tsv_file:
+        tsv_writer = csv.writer(tsv_file, delimiter='\t', lineterminator='\n')
+        tsv_writer.writerow(["participant_id"])
+        for item in participants:
+            tsv_writer.writerow(item)
+
+    # Create samples.tsv
+    with open(output_data + '/samples.tsv', 'w') as tsv_file:
+        tsv_writer = csv.writer(tsv_file, delimiter='\t', lineterminator='\n')
+        tsv_writer.writerow(["participant_id", "sample_id", "sample_type"])
+        for item in samples:
+            tsv_writer.writerow(item)
+
+    # Create dataset_description.json
+    dataset_description = {"Name": "data_axondeepseg_tem",
+                           "BIDSVersion": "1.6.0 - BEP031 v0.0.4",
+                           "License": "MIT"
+                           }
+
+    with open(output_data + '/dataset_description.json', 'w') as json_file:
+        json.dump(dataset_description, json_file, indent=4)
+
+    # Create participants.json
+    data_json = {"participant_id": {
+        "Description": "Unique Participant ID",
+        "LongName": "Participant ID"
+    }}
+
+    with open(output_data + '/participants.json', 'w') as json_file:
+        json.dump(data_json, json_file, indent=4)
+
+    # Create samples.json
+    data_json = {"participant_id": {
+        "Description": "Unique Participant ID",
+        "LongName": "Participant ID"
+         },
+        "sample_id": {
+            "Description": "Sample ID",
+            "LongName": "Sample ID"
+        },
+        "sample_type": {
+            "Description": "Sample type",
+            "LongName": "Sample type"
+        }
+    }
+
+    with open(output_data + '/samples.json', 'w') as json_file:
+        json.dump(data_json, json_file, indent=4)
+
+    # Create README
+    with open(output_data + '/README', 'w') as readme_file:
+        readme_file.write('Dataset for data_axondeepseg_tem.')
```

Comment:
This README isn't very descriptive? What's data_axondeepseg_tem? Is there a link to the project you can include here?